docs/reference/modules/discovery/zen.asciidoc

```diff
@@ -75,7 +75,7 @@ The `discovery.zen.minimum_master_nodes` sets the minimum
 number of master eligible nodes that need to join a newly elected master in order for an election to
 complete and for the elected node to accept its mastership. The same setting controls the minimum number of
 active master eligible nodes that should be a part of any active cluster. If this requirement is not met the
-active master node will step down and a new master election will be begin.
+active master node will step down and a new master election will be begun.
```

Comment:
I think this should be "active master node will step down and a new master election will *begin*".